scripts/launch/connect-to-cluster/mainnet-exchange.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+readonly CLUSTER=mainnet
+
+flush_wallet=""
+
+if [[ "$1" == "-c" ]]; then
```

Comment:
I believe now that it's safer to rename this argument, because the same argument in `mainnet.sh` just completely deletes all databases and keyfiles (and if customer by accident uses the wrong script (or we provide the wrong instruction) without having a proper backup, that would be a disaster).

Reply:
I agree, -c is likely to cause confusion.